index.js

```diff
@@ -1,23 +1,193 @@
-console.log("hello there")
-
-var test = {
-	escape: function(html) {
-		return String(html)
-			.replace(/&/g, '&amp;')
-			.replace(/"/g, '&quot;')
-			.replace(/'/g, '&#39;')
-			.replace(/</g, '&lt;')
-			.replace(/>/g, '&gt;');
-	},
-
-	unescape: function(html) {
-		return String(html)
-			.replace(/&amp;/g, '&')
-			.replace(/&quot;/g, '"')
-			.replace(/&#39;/g, '\'')
-			.replace(/&lt;/g, '<')
-			.replace(/&gt;/g, '>');
-	}
+
+// placeholder for now - should talk when merged to properly integrate this with 
+// existing "board" and "players" implementations
+function Board(players) { 
```

Comment:
Is this the way to make a class in JS? having it be a function feels weird but that's could be my lack of knowledge.